R/engine.R

```diff
@@ -738,7 +738,19 @@ eng_sxss = function(options) {
   }
 
   engine_output(options, options$code, final_out)
+}
 
+eng_bslib = function(options) {
+  if (!loadable("bslib")) {
+    stop("The {bslib} package must be installed in order for the {bslib} knitr engine to work.")
+  }
+  if (!is.null(options$engine.opts$sass_fun)) {
+    stop("The {bslib} knitr engine does not allow for customization of the Sass compilation function.", call. = FALSE)
+  }
+  func <- sass::sass_partial
+  formals(func)$bundle <- quote(bslib::bs_global_get())
+  options$engine.opts$sass_fun <- func
+  eng_sxss(options)
```

Comment:
I think style is `=` instead of `<-` from other engine
```suggestion
  func = sass::sass_partial
  formals(func)$bundle = quote(bslib::bs_global_get())
  options$engine.opts$sass_fun = func
  eng_sxss(options)
```